sdk/appcenter-crashes/src/main/java/com/microsoft/appcenter/crashes/Crashes.java

```diff
@@ -206,13 +206,28 @@ static synchronized void unsetInstance() {
 
     /**
      * Track an exception.
-     * TODO the backend does not support that service yet, will be public method later.
      *
      * @param throwable An exception.
      */
     @SuppressWarnings("SameParameterValue")
-    static void trackException(@NonNull Throwable throwable) {
-        getInstance().queueException(throwable);
+    public static void trackException(@NonNull Throwable throwable) {
```

Comment:
oh no don't make it public yet, the public visibility is only for Xamarin (And wrapper sdk exception manager), restore package private and the TODO